lib/vets/attributes.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Vets
+  module Attributes
+    def self.included(base)
+      base.extend(ClassMethods)
+      base.instance_variable_set(:@attributes, {})
+    end
+
+    module ClassMethods
+
+      # rubocop:disable ThreadSafety/InstanceVariableInClassMethod
+      def attributes
+        @attributes
```

Comment:
I think you can just inline disable?
```suggestion
        @attributes # rubocop:disable ThreadSafety/InstanceVariableInClassMethod
```